dnf5/commands/download/download.cpp

```diff
@@ -161,10 +182,25 @@ void DownloadCommand::run() {
     }
 
     if (url_option->get_value()) {
+        // If no urlprotocols are specified, then any urlprotocol is acceptable
```

Comment:
Not any protocol. The four protocols listed in `urlprotocol_valid_options`.

Reply:
Wouldn't be easier to error on `--urlprotocol` option without an argument? If one does not want to limit the protocols, he simply does not pass this option.

Reply:
It currently does error if there's no value passed into the --urlprotocol option. I believe this is because the set_has_value attribute is set to true
<img width="741" alt="image" src="https://github.com/rpm-software-management/dnf5/assets/44981122/e552c1ce-80cb-40cd-87ee-37ad23f686ba">